src/web/Html.tsx

```diff
@@ -149,14 +149,20 @@ export const Html = React.forwardRef(
     const raycaster = useThree(({ raycaster }) => raycaster)
 
     const [el] = React.useState(() => document.createElement(as))
-    const root = React.useMemo(() => ReactDOM.createRoot(el), [el])
+    const root = React.useRef<ReactDOM.Root>()
     const group = React.useRef<Group>(null!)
     const oldZoom = React.useRef(0)
     const oldPosition = React.useRef([0, 0])
     const transformOuterRef = React.useRef<HTMLDivElement>(null!)
     const transformInnerRef = React.useRef<HTMLDivElement>(null!)
     const target = portal?.current ?? gl.domElement.parentNode
 
+    React.useEffect(() => {
```

Comment:
wont this create a new root every render?

Reply:
Yeah, do we have a way to check if an element is already a reactDom root?
Otherwise maybe a simple check on the ref would work - assuming unmount also sets it back to undefined

```
if (!root.current) {
   ...
}
```

Reply:
The problem before was it was losing the root when re-rendering after a transform prop change.
So another thing could be manually disposing of the root only when it's no longer necessary, so still keeping the code in a useEffect but only unmounting if the root.current is defined AND transform is now false